Sources/Transmute/Package+testModules.swift

```diff
@@ -13,8 +13,24 @@ import Utility
 
 extension Package {
     func testModules() throws -> [TestModule] {
-        return try walk(path, "Tests", recursively: false).filter{ $0.isDirectory }.map { dir in
-            return TestModule(basename: dir.basename, sources: try self.sourcify(dir))
+        let (directories, files) = walk(path, "Tests", recursively: false).partition{ $0.isDirectory }
+        
+        if (directories.count > 0) {
+            return try directories
```

Comment:
It should be a invalid layout if there are files except `LinuxMain.swift` in root as well as directories (not in excludes)